pymc_experimental/tests/statespace/utilities/test_helpers.py

```diff
@@ -20,7 +20,7 @@
 
 def load_nile_test_data():
     nile = pd.read_csv("pymc_experimental/tests/statespace/test_data/nile.csv", dtype={"x": floatX})
-    nile.index = pd.date_range(start="1871-01-01", end="1970-01-01", freq="AS-Jan")
+    nile.index = pd.date_range(start="1871-01-01", end="1970-01-01", freq="YS-JAN")
```

Comment:
Tests failing because of this?

Reply:
Yeah, I guess pandas just did a deprecation warning. I was blindsided by it.

Reply:
But tests are still failing